src/CMakeLists.txt

```diff
@@ -206,6 +211,11 @@ if( EXIV2_ENABLE_NLS )
     target_compile_definitions(exiv2lib PUBLIC EXV_LOCALEDIR="/../${CMAKE_INSTALL_LOCALEDIR}")
 endif()
 
+if( EXIV2_ENABLE_HEIF )
+    target_include_directories(exiv2lib_int PRIVATE ${HEIF_INCLUDE_DIR})
```

Comment:
The new files are part of the target `exiv2lib`. I think you would need to replace `exiv2lib_int` here by `exiv2lib`